pkg/apis/rke.cattle.io/v1/cluster.go

```diff
@@ -70,7 +70,7 @@ type DrainOptions struct {
 	// (even when set to true, kubectl won't delete pods - so setting default to true)
 	IgnoreDaemonSets *bool `json:"ignoreDaemonSets"`
 	// IgnoreErrors Ignore errors occurred between drain nodes in group
-	IgnoreErrors bool
+	IgnoreErrors bool `json:"ignoreErrors"`
```

Comment:
Looks like a sane change, but curious if it's related to the issue at hand as we're not setting `IgnoreErrors` in the test below? Maybe we should?